service/src/test/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsServiceTest.java

```diff
@@ -457,43 +468,48 @@ void shouldNotSkipPaymentsWhenApplyingForAnOtherOrder() {
 
     @Nested
     class PostSubmitProcessing {
+        private static final CaseData CASE_DATA = CaseData.builder().court(COURT_1).build();
 
         @Test
         void shouldSetSupplementsToEmptyListIfNonePresent() {
             C2DocumentBundle bundle = C2DocumentBundle.builder()
                 .document(DOCUMENT)
                 .supplementsBundle(List.of())
                 .build();
-            C2DocumentBundle converted = underTest.convertC2Bundle(bundle);
+            C2DocumentBundle converted = underTest.convertC2Bundle(bundle, CASE_DATA);
 
             assertThat(converted.getSupplementsBundle()).isEmpty();
             assertThat(converted.getSupplementsBundle()).isNotNull();
         }
 
         @Test
-        void shouldConvertC2SupplementToPdf() {
+        void shouldSealC2Document() {
             C2DocumentBundle bundle = C2DocumentBundle.builder()
                 .document(DOCUMENT)
                 .supplementsBundle(wrapElementsWithRandomUUID(Supplement.builder()
                     .document(SUPPLEMENT_DOCUMENT)
                     .build()))
                 .build();
-            C2DocumentBundle converted = underTest.convertC2Bundle(bundle);
+            C2DocumentBundle converted = underTest.convertC2Bundle(bundle, CASE_DATA);
 
+            assertThat(converted.getDocument())
+                .isEqualTo(SEALED_DOCUMENT);
             assertThat(converted.getSupplementsBundle().get(0).getValue().getDocument())
                 .isEqualTo(CONVERTED_SUPPLEMENT_DOCUMENT);
         }
 
         @Test
-        void shouldConvertOtherSupplementToPdf() {
+        void shouldSealOtherDocumentIf() {
```

Comment:
maybe check the name here ... if what? :)

Reply:
fixed to shouldSealOtherDocument